pkg/server/plugin/upstreamauthority/spire/spire_test.go

```diff
@@ -326,9 +326,10 @@ func TestPublishJWTKey(t *testing.T) {
 	require.NotNil(t, upstreamJwtKeys)
 
 	require.Len(t, upstreamJwtKeys, 3)
-	assert.Equal(t, upstreamJwtKeys[0].Kid, "C6vs25welZOx6WksNYfbMfiw9l96pMnD")
-	assert.Equal(t, upstreamJwtKeys[1].Kid, "gHTCunJbefYtnZnTctd84xeRWyMrEsWD")
-	assert.Equal(t, upstreamJwtKeys[2].Kid, "kid-2")
+	keyIDs := []string{upstreamJwtKeys[0].Kid, upstreamJwtKeys[1].Kid, upstreamJwtKeys[2].Kid}
+	assert.Contains(t, keyIDs, "C6vs25welZOx6WksNYfbMfiw9l96pMnD")
+	assert.Contains(t, keyIDs, "gHTCunJbefYtnZnTctd84xeRWyMrEsWD")
+	assert.Contains(t, keyIDs, "kid-2")
```

Comment:
so order is no longer important?

Reply:
@MarcosDY actually, this is happening due to changes on fake_handlers_test; this sorting break is not happening in the real implementation, I'm going to update it!